engine/polyglot-api/src/main/java/org/enso/polyglot/RuntimeOptions.java

```diff
@@ -16,7 +16,7 @@ public class RuntimeOptions {
       OptionDescriptor.newBuilder(PROJECT_ROOT_KEY, PROJECT_ROOT).build();
 
   public static final String STRICT_ERRORS = optionName("strictErrors");
-  public static final OptionKey<Boolean> STRICT_ERRORS_KEY = new OptionKey<>(false);
+  public static final OptionKey<Boolean> STRICT_ERRORS_KEY = new OptionKey<>(true);
```

Comment:
With the most recent change 747fd5c it is possible to run `sbt runtime-integration-tests/test` succeeds with `false` as well as `true` default value of `STRICT_ERRORS_KEY` - e.g. all our tests override the default or don't care - except the single test from the Truffle TCK:
```
[error] Test org.enso.interpreter.test.tck.EnsoTckInvalidSyntaxTest.testInvalidSyntax
  [enso::unknown_x.enso] failed: java.lang.AssertionError: Expected exception.
[error]     at com.oracle.truffle.tck.tests.InvalidSyntaxTest.testInvalidSyntax(:116)
```
which uses the default - as the `org.graalvm.polyglot.Context` isn't created by us, but rather by the TCK itself and doesn't specify value of `STRICT_ERRORS_KEY`.